### PR TITLE
feat: add signup form

### PR DESCRIPTION
**Pull request by theetrain**
This is for new members to learn more about our conduct before registering.

Screen Recording 2020-06-15 at 09 45 PM

Screen Recording 2020-06-15 at 09 48 PM

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -14,4 +14,3 @@ h2 {
   font-size: 2.8rem;
   margin: 0;
 }
-
```
Comment (ahmadnassri) [inline _sass/_base.scss:17]: THE HORROR!!!! REJECTING THIS PR NOW!

**Review by ahmadnassri — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -47,44 +47,140 @@
...
+          <div>
+            <label>
+              <input autocomplete="off" type="checkbox" id="wontAbuse" />
+              I will treat others with mutual respect and will not abuse others
+            </label>
+          </div>
+          <div>
+            <label>
+              <input autocomplete="off" type="checkbox" id="wontSpam" />
+              I will not spam, and will read channel descriptions before posting
```
Comment (cr03) [inline index.html:70]:
```suggestion
              I will not post anything related to my pecuniary interests without written approval from the administrators, and only then in the #showcase channel. 
```

**Review by cr03 — CHANGES_REQUESTED** [review]
I just have a minor ask to clarify this language. Cause really, what is "spam"? I say: tie it to any pecuniary interest (you're doing it for your job or your company.)

**Comment by adrianmaurer** [comment]
Can I suggest adding a line constituting my powers and membership?

**Pull request merged**